AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -389,4 +392,10 @@ - (BOOL)didReceiveRemoteNotification:(NSDictionary *)userInfo fromUserNotificati
   return NO;
 }
 
+- (void)authTokenContext:(__unused MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(__unused NSString *)authToken {
+  if (self.pushToken) {
```

Comment:
This will execute if Push is disabled.

Reply:
We decided that the best place to add / remove delegate is in `applyEnabledState`